OrbitGl/TimeGraph.cpp

```diff
@@ -1099,7 +1099,9 @@ int TimeGraph::FindMovingTrackIndex() {
 }
 
 void TimeGraph::UpdateTracks(uint64_t min_tick, uint64_t max_tick, PickingMode picking_mode) {
-  float current_y = -layout_.GetSchedulerTrackOffset();
+  // As we draw the track tab on top of current_y coordinate, so we need this extra space to not
+  // draw anything on positive y-coordinates.
```

Comment:
nit. The wording is weird here. Either remove "so" or use a simpler expression: 

// Make sure track tab fits in the viewport. 

I'd go with the latter.

Reply:
Done! Thanks :)